src/services/speakers/speakers-schema.ts

```diff
@@ -1,18 +1,6 @@
 import { Schema } from "mongoose";
-import { z } from "zod";
 import { v4 as uuidv4 } from "uuid";
 
-// Zod schema for speaker
-export const SpeakerValidator = z.object({
-    speakerId: z.coerce.string().default(() => uuidv4()),
```

Comment:
this shouldnt be needed - also make changes to speaker, so that we have the following:

GET /speakers/:SPEAKERID

PUT /speakers/:SPEAKERID

POST /speakers/ (create speakerid)

Reply:
I think those endpoints were added already ?